iceoryx_utils/platform/win/include/iceoryx_utils/platform/mman.hpp

```diff
@@ -17,7 +17,7 @@
 #include "iceoryx_utils/platform/fcntl.hpp"
 #include "iceoryx_utils/platform/types.hpp"
 #include "iceoryx_utils/platform/unistd.hpp"
-#include "iceoryx_utils/platform/win32-error.hpp"
+#include "iceoryx_utils/platform/win32_errorHandling.hpp"
```

Comment:
what about win64? i would remove the 32 from the name

Reply:
its because of win32 platform definition

Reply:
The windows platform always refers to itself as win32. I wanted to stay in the domain language and therefore it is win32